golioth_sdk/dispatch.c

```diff
@@ -5,7 +5,7 @@
  */
 
 #include <zephyr/logging/log.h>
-LOG_MODULE_REGISTER(dispatch, LOG_LEVEL_DBG);
+LOG_MODULE_REGISTER(glth_dispatch, LOG_LEVEL_DBG);
```

Comment:
Using `glth_` instead of `golioth_` may be too defensive, given that I don't think we currently envision a scenario where the `golioth_sdk` portion of this repository is used alongside the existing Golioth Firmware SDK. Using the short-hand may also just be nice across the board though. Open to discussion!